src/services/clients/dissolutionApi.client.ts

```diff
@@ -30,4 +33,24 @@ export class DissolutionApiClient {
     )
     return response.data
   }
+
+  public async getDissolution(token: string, companyNumber: string): Promise<Optional<DissolutionGetResponse>> {
+    const response: Optional<AxiosResponse<DissolutionGetResponse>> = await this.axios.get(
+      `${this.DISSOLUTIONS_API_URL}/dissolution-request/${companyNumber}`,
+      {
+        headers: {
+          Authorization: 'Bearer ' + token,
+          'Content-Type': 'application/json',
+          'Accept': 'application/json',
+        }
+      }
```

Comment:
Looks to be a lot of repetition between this and the createDissolution method. Can we introduce some reusable functions and constants? Maybe try declaring the URLs in `private readonly CONSTANT_NAME: string = ...` at the top of this class